client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -24,17 +24,14 @@ const debug = debugFactory( 'calypso:domains:registrant-extra-info' );
 // and we don't particularly want to make the parent remember all our fields
 // so we use these values to plug missing.
 const emptyValues = {
-	countryOfBirth: '',
-	placeOfBirth: '',
-	postalCodeOfBirth: '',
 	registrantVatId: '',
 	sirenSiret: '',
 	trademarkNumber: '',
 };
 
 class RegistrantExtraInfoFrForm extends React.PureComponent {
 	static propTypes = {
-		countriesList: PropTypes.object.isRequired,
+		countriesList: PropTypes.object,
```

Comment:
Ha. Looks like yes. I started to do that but then left it because it isn't necessarily `fr` specific in the way it gets passed down (from general domain details -> add'l TLD details container -> specific TLD details form).  But I looked again after your comment and `ca` doesn't use it and I don't see `de` or `in` needing it either.

So, will yank it out all the way up the chain and will push up in a few.